account_banking_pain_base/models/account_payment_order.py

```diff
@@ -540,7 +519,34 @@ def generate_address_block(self, parent_node, partner, gen_args):
                     70,
                     gen_args=gen_args,
                 )
-
+            if (
+                gen_args.get("pain_flavor").startswith("pain.001.001.")
+                or gen_args.get("pain_flavor").startswith("pain.008.001.")
```

Comment:
Which one is `pain.008`?

Reply:
@pedrobaeza 
I don't know if I understand your question correctly.
pain.008 is for SEPA Direct Debit.